deployments/helm-chart/Chart.yaml

```diff
@@ -2,6 +2,7 @@ name: nginx-ingress
 version: edge
 appVersion: edge
 apiVersion: v1
+kubeVersion: ">= 1.15.0"
```

Comment:
In the main.go, we support 1.14. Here in helm, we don't support it. Was it intentional?  🤔